instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -391,12 +404,32 @@ def _populate_span(
         if self._db_api_integration.capture_parameters and len(args) > 1:
             span.set_attribute("db.statement.parameters", str(args[1]))
 
-    def get_operation_name(self, cursor, args):  # pylint: disable=no-self-use
+    def get_span_name(self, cursor, args):
+        operation_name = self.get_operation_name(cursor, args)
+        statement = self.get_statement(cursor, args)
+        collection_name = CursorTracer.get_collection_name(statement)
+        return " ".join(
+            name for name in (operation_name, collection_name) if name
+        )
```

Comment:
Do these changes also implement these parts of the [spec](https://opentelemetry.io/docs/specs/semconv/database/database-spans/#name)?

> If there is no (low-cardinality) db.operation.name available, database span names SHOULD be [{target}](https://opentelemetry.io/docs/specs/semconv/database/database-spans/#target-placeholder).
> 
> If neither {db.operation.name} nor {target} are available, span name SHOULD be {db.system}.

I think Yes for the first point, would be good to see some test cases for that as well.